src/Symfony/Component/Validator/Constraints/YamlValidator.php

```diff
@@ -59,5 +64,28 @@ public function validate(mixed $value, Constraint $constraint): void
         } finally {
             restore_error_handler();
         }
+
+        if (isset($data) && null !== $constraint->configClass) {
+            if (!interface_exists(ConfigurationInterface::class)) {
+                throw new LogicException('To use "configurationClass" option, you need to install "symfony/config" dependency.');
```

Comment:
```suggestion
                throw new LogicException('The "configurationClass" option requires the "%s" interface. Try running "composer require symfony/config".', ConfigurationInterface::class);
```

To match other occurrences of messages of this type